### ISSUE TITLE
extension function not reported if erased type has same signature

### ISSUE THREAD

**Issue by robstoll** [Kotlin/dokka#408]
I have the following functions:
```
fun <T : CharSequence> Assert<T>.isEmpty()
fun <T : Collection<*>> Assert<T>.isEmpty()
fun <T : Map<*, *>> Assert<T>.isEmpty()
```
Only the function for CharSequence is reported, the one for Collection and Map aren't. If I change the one of Map to the following
```
fun Assert<out Map<*, *>>.isEmpty()
```
then it is reported as well. So I guess it is a problem related to type erasure or similar. Just a wild guess, the upper bound is ignored which makes all functions to look the same `Assert<Any>.isEmpty()`

**Comment by kamilok1965** [comment]
This is probably a duplicate of https://github.com/Kotlin/dokka/issues/408

**Comment by robstoll** [comment]
@kamilok1965 a duplicate of itself? :wink:

**Comment by kamilok1965** [comment]
Oh, this was supposed to be in another issue, which I cannot find right now

**Comment by kamildoleglo** [comment]
Fixed while rewriting dokka, fix confirmed in `1.4.10.2`
